RemDB/dist/metadata.json

```diff
@@ -0,0 +1,18 @@
+{
+  "author": {
+    "name": "Kevin",
+    "id": "698950867184779404"
+  },
+  "service": "RemDB",
+  "description": {
+    "en": "RemDB is a Player- and Clandatabase from a private server (S4 Remnants) of the game S4 League.",
+    "de": "RemDB ist eine Spieler- und Clandatenbank für einen privaten Server (S4 Remnants) vom Spiel S4 League."
+  },
+  "url": ["remdb.net"],
+  "version": "1.0.0",
+  "logo": "https://kevin.s-ul.eu/AHQBFYlq.png",
+  "thumbnail": "https://kevin.s-ul.eu/AHQBFYlq.png",
```

Comment:
upload to imgur
and use a picture of the website as thumbnail

Reply:
Ah my bad. Pushed the fix.